src/PostalCodes.UnitTests/CountrySpecificPostalCodes/NLPostalCodeManualTests.cs

```diff
@@ -33,13 +33,13 @@ public void Successor_ValidInput_ReturnsCorrectPostalCode(string postalCode, str
             Assert.AreEqual(postalCodeSuccessor, (new NLPostalCode(postalCode)).Successor.ToString());
         }
 
-        [Test]
-        [TestCase("0000 AA")]
```

Comment:
Could you add these test cases to the tests with invalid postal codes? Also add the `SA|SD|SS` cases please